src/contracts/ContractForm.tsx

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import {  useSelector } from "react-redux";
+import { ExplainSideColumn } from '../components/ExplainSideColumn';
+import { TextInput } from './../form/TextInput';
+import { CheckBox } from './../form/CheckBox';
+import { CancelButton } from '../form/CancelButton';
+import { useForm, Values } from '../form/useForm';
+import { minLength } from '../form/validators';
+import { getContractForSelectedOrganisation, getUsage } from '../reducers';
+import { helpTextContractView } from '../utils/help_texts/helpTextContractView';
+import formStyles from './../styles/Forms.module.css';
+import agreementIcon from "../images/agreement.svg";
+import { TextArea } from './../form/TextArea';
+import UsagePieChart from './../components/UsagePieChart';
+
+
+
+export const ContractForm = () => {
+
+  const contractObjApi = useSelector(getContractForSelectedOrganisation);
+  const usageObj = useSelector(getUsage);
+
+  const initialValues = {
+  };
+
+  const onSubmit = (values: Values) => {};
+
+  const {
+    values,
+    triedToSubmit,
+    handleValueChange,
+    handleInputChange,
+    fieldOnFocus,
+    handleFocus,
+    handleBlur,
+    handleSubmit,
+    handleReset,
+    clearInput,
+  } = useForm({initialValues, onSubmit});
+
+  return (
+    <ExplainSideColumn
+      imgUrl={agreementIcon}
+      imgAltDescription={"Contract Icon"}
+      headerText={"Contract"}
+      explanationText={helpTextContractView[fieldOnFocus] || helpTextContractView['default']}
+      backUrl={"/management"}
+      fieldName={fieldOnFocus}
+    >
+      <form
+        className={formStyles.Form}
+        onSubmit={handleSubmit}
+        onReset={handleReset}
+      >
+        <span className={`${formStyles.FormFieldTitle} ${formStyles.FirstFormFieldTitle}`}>
+          1: General
+        </span>
+        <TextInput
+          title={'Start dat contract'}
+          name={'start_date_contract'}
+          value={(new Date(contractObjApi.start).toLocaleDateString())}
+          valueChanged={handleInputChange}
+          clearInput={clearInput}
+          validated={!minLength(2, values.name)}
+          errorMessage={minLength(2, values.name)}
+          triedToSubmit={triedToSubmit}
```

Comment:
I think these props can be removed since this is only a read-only field so no validation is needed (only use `validated`)

Reply:
done